gitgud/test_commands.py

```diff
@@ -6,13 +6,13 @@
 
 def test_load(gg):
     load_tests = [
-        ('git gud load 1', all_skills[0][0]),
-        ('git gud load rampup', all_skills["rampup"][0]),
-        ('git gud load 2 detaching', all_skills[1]["detaching"]),
-        ('git gud load rampup 4', all_skills["rampup"][3]),
-        ('git gud load 3-octopus', all_skills[2]["octopus"]),
-        ('git gud load basics-4', all_skills["basics"][3]),
-        ('git gud load -2', all_skills["basics"][1])
+        ('git gud load 1', all_skills["1"]["1"]),
+        ('git gud load rampup', all_skills["rampup"]["1"]),
+        ('git gud load 2 detaching', all_skills["2"]["detaching"]),
+        ('git gud load rampup 4', all_skills["rampup"]["4"]),
+        ('git gud load 3-octopus', all_skills["3"]["octopus"]),
+        ('git gud load basics-4', all_skills["basics"]["4"]),
+        ('git gud load -2', all_skills["basics"]["2"])
```

Comment:
I changed this test case to refer to "rampup" just to make sure that it's not defaulting to the first level